src/bcs/serializable/scriptRaw.ts

```diff
@@ -0,0 +1,30 @@
+import { Serializer, Serializable } from "../serializer";
+import { Deserializer } from "../deserializer";
+import { HexInput } from "../../types";
+import { Hex } from "../../core/hex";
+import { TransactionArgument } from "../../transactions/instances/transactionArgument";
+
+export class Raw extends Serializable implements TransactionArgument {
```

Comment:
Either rename the class name or the file name
```suggestion
export class ScriptRaw extends Serializable implements TransactionArgument {
```